src/test/java/net/minecraftforge/debug/world/RaidEnumTest.java

```diff
@@ -0,0 +1,19 @@
+package net.minecraftforge.debug.world;
+
+import net.minecraft.entity.EntityType;
+import net.minecraft.world.raid.Raid;
+import net.minecraftforge.fml.common.Mod;
+
+@Mod("raid_enum_test")
+public class RaidEnumTest 
+{
+    private static final boolean ENABLE = false;
+	
+    public RaidEnumTest()
+    {
+     if (ENABLE)
+     {
+       Raid.WaveMember.create("thebluemengroup", EntityType.ILLUSIONER, new int[]{0, 5, 0, 1, 0, 1, 0, 2});
+     }
+    }
```

Comment:
This class's code style is all kinda of messed up.... Have you looked at it? Honestly you could just skip this tho, as the IExtensibleEnable should explode if you did it wrong.